app/views/schools/advice/_nav.html.erb

```diff
@@ -1,23 +1,39 @@
-<%= component 'page_nav', name: t('advice_pages.nav.name'), icon: 'home', href: school_advice_path(@school), options: { match_controller: true } do |c| %>
+<%= component 'page_nav', name: t('advice_pages.nav.name'),
+                          icon: 'home',
+                          href: school_advice_path(@school),
+                          options: { match_controller: true } do |c| %>
   <% c.with_section do |s| %>
-    <% s.with_item(name: t('advice_pages.nav.pages.total_energy_use'), href: school_advice_total_energy_use_path(@school)) %>
+    <% s.with_item(name: t('advice_pages.nav.pages.total_energy_use'),
+                   href: school_advice_total_energy_use_path(@school)) %>
   <% end %>
-  <% c.with_section(name: t('advice_pages.nav.sections.electricity'), icon: 'bolt', classes: 'electric-section', visible: @school.has_electricity?) do |s| %>
+  <% c.with_section(name: t('advice_pages.nav.sections.electricity'),
+                    icon: 'bolt',
+                    classes: 'electric-section',
+                    visible: @school.has_electricity?) do |s| %>
     <% sort_by_label(advice_pages.where(fuel_type: :electricity)).each do |ap| %>
       <% s.with_item(name: translated_label(ap), href: advice_page_path(@school, ap), classes: 'electric-item') %>
     <% end %>
   <% end %>
-  <% c.with_section(name: t('advice_pages.nav.sections.gas'), icon: 'fire', classes: 'gas-section', visible: @school.has_gas?) do |s| %>
+  <% c.with_section(name: t('advice_pages.nav.sections.gas'),
+                    icon: 'fire',
+                    classes: 'gas-section',
+                    visible: @school.has_gas?) do |s| %>
     <% sort_by_label(advice_pages.where(fuel_type: :gas)).each do |ap| %>
       <% s.with_item(name: translated_label(ap), href: advice_page_path(@school, ap), classes: 'gas-item') %>
     <% end %>
   <% end %>
-  <% c.with_section(name: t('advice_pages.nav.sections.storage_heater'), icon: 'window-maximize', classes: 'storage-section', visible: @school.has_storage_heaters?) do |s| %>
+  <% c.with_section(name: t('advice_pages.nav.sections.storage_heater'),
+                    icon: 'window-maximize',
+                    classes: 'storage-section',
+                    visible: @school.has_storage_heaters?) do |s| %>
     <% sort_by_label(advice_pages.where(fuel_type: :storage_heater)).each do |ap| %>
       <% s.with_item(name: translated_label(ap), href: advice_page_path(@school, ap), classes: 'storage-item') %>
     <% end %>
   <% end %>
-  <% c.with_section(name: t('advice_pages.nav.sections.solar_pv'), icon: 'sun', classes: 'solar-section', visible: @school.has_electricity?) do |s| %>
+  <% c.with_section(name: t('advice_pages.nav.sections.solar_pv'),
+                    icon: 'sun',
+                    classes: 'solar-section',
+                    visible: @school.has_solar_pv?) do |s| %>
```

Comment:
The actual change here is changing this:
`visible: @school.has_electricity?`
to
`visible: @school.has_solar_pv?`

Reply:
No, because there are actually two version of the solar page: If a school has solar we report on how much they've generated. If they have electricity but no solar, then we estimate impacts of adding solar.

Reply:
Ahh didn't realise (and that's a good feature), thanks @ldodds 👍